goshopify.go

```diff
@@ -241,12 +241,16 @@ func (c *Client) Do(req *http.Request, v interface{}) error {
 	}
 
 	if callLimit, ok := resp.Header[callLimitHeader]; ok {
+		logMessage := fmt.Sprintf("%s: %s", callLimitHeader, callLimit[0])
 		// Extract the current bucket capacity from the call limit response header
 		bucketCapacity, err = strconv.Atoi(strings.Split(callLimit[0], "/")[0])
 		if err != nil {
-			return err
+			logMessage = "Unable to parse the call limit header. " + logMessage
+			// set bucket capacity to the threshold + 1 to throttle requests until call limit header can be parsed
+			bucketCapacity = c.app.BucketThreshold + 1
```

Comment:
`c.app.BucketThreshold` is never set. So bucketCapacity will just be 1 when no header exists. correct?

Reply:
Also, `c.app.HeavyLoadDelay` is never set. so line 230 is never actually going to take effect.

Reply:
the c.appBucketThreshold and c.app.HeavyLoadDelay are set from the outside by clients. If the clients don't set them then no auto-throttling is happening (same behavior as before). This is by design.  See the PRs for all the services that use go-shopify